src/pudl/transform/ferc1.py

```diff
@@ -3705,7 +3705,7 @@ def process_xbrl(
         )
 
 
-class OtherRegulatoryLiabilitiesFerc1(Ferc1AbstractTableTransformer):
+class OtherRegulatoryLiabilitiesFerc1TableTransformer(Ferc1AbstractTableTransformer):
```

Comment:
There was some code in the ferc-debug-etl notebook that gathered transform classes by looking for the "Ferc1TableTransformer" substring. It missed this class so I renamed it.